### ISSUE TITLE
Run lighthouse accessibility checks and create issues for findings

### ISSUE THREAD

**Issue by mcharg** [ONRR/onrr.gov-site#1954]

**Comment by mcharg** [comment]
Also check with screen reader.

**Comment by mcharg** [comment]
Pages:

- [ ] Home
- [ ] Getting Started Home
- [ ] How Leasing Works
- [ ] New Reporter Checklist
- [ ] System Access
- [ ] New Reporter Training
- [ ] Reporting Home
- [ ] Oil & Gas Production
- [ ] Oil & Gas Revenue
- [ ] Solid Minerals
- [ ] Geothermal
- [ ] References (Home)
- [ ] Statutes
- [ ] FOGRMA Amendments
- [ ] Rulemakings
- [ ] Regulation Identifier Number 1012-AA29
- [ ] Regulation Identifier Number 1012-AA27
- [ ] Regulation Identifier Number 1012-AA26
- [ ] Regulation Identifier Number 1012-AA25
- [ ] Regulation Identifier Number 1012-AA24
- [ ] Regulation Identifier Number 1012-AA23
- [ ] Regulation Identifier Number 1012-AA22
- [ ] Regulation Identifier Number 1012-AA21
- [ ] Regulation Identifier Number 1012-AA20
- [ ] Regulation Identifier Number 1012-AA17
- [ ] Regulation Identifier Number 1012-AA15
- [ ] Regulation Identifier Number 1012-AA14
- [ ] Regulation Identifier Number 1012-AA13
- [ ] Regulation Identifier Number 1012-AA11
- [ ] Regulation Identifier Number 1010-AD70
- [ ] Regulation Identifier Number 1012-AA08
- [ ] Regulation Identifier Number 1012-AA05
- [ ] Regulation Identifier Number 1012-AA04
- [ ] Regulation Identifier Number 1012-AA03
- [ ] Regulation Identifier Number 1012-AA02
- [ ] Regulation Identifier Number 1010-AD32
- [ ] Regulation Identifier Number 1010-AD28
- [ ] Regulation Identifier Number 1010-AD22
- [ ] Regulation Identifier Number 1010-AD20
- [ ] Regulation Identifier Number 1010-AD05
- [ ] Regulation Identifier Number 1010-AD04
- [ ] Regulation Identifier Number 1010-AD01
- [ ] Regulation Identifier Number 1010-AD00
- [ ] Regulation Identifier Number 1010-AC86
- [ ] Regulation Identifier Number 1010-AC72
- [ ] Regulation Identifier Number 1010-AC70
- [ ] Regulation Identifier Number 1010-AC59
- [ ] Regulation Identifier Number 1010-AC51
- [ ] Regulation Identifier Number 1010-AC40
- [ ] Regulation Identifier Number 1010-AC38
- [ ] Regulation Identifier Number 1010-AC30
- [ ] Regulation Identifier Number 1010-AC25
- [ ] Regulation Identifier Number 1010-AC21
- [ ] Regulation Identifier Number 1010-AC08
- [ ] Regulation Identifier Number 1010-AC06
- [ ] Regulation Identifier Number 1010-AC00
- [ ] Regulation Identifier Number 1010-AB57
- [ ] Regulation Identifier Number 1010-AB17
- [ ] 1989 Coal Rule: Revision of Coal Product Valuation Regulations and Related Topics
- [ ] Federal Register Notices
- [ ] Information Collection Requests
- [ ] OMB Control Number 1012-0001
- [ ] OMB Control Number 1012-0002
- [ ] OMB Control Number 1012-0003
- [ ] OMB Control Number 1012-0004
- [ ] OMB Control Number 1012-0005
- [ ] OMB Control Number 1012-0006
- [ ] OMB Control Number 1012-0008
- [ ] OMB Control Number 1012-0009
- [ ] OMB Control Number 1012-0010
- [ ] OMB Control Number 1012-0012
- [ ] OMB Control Number 1012-0007
- [ ] OMB Control Number 1010-0090
- [ ] OMB Control Number 1010-0140
- [ ] OMB Control Number 1010-0169
- [ ] OMB Control Number 1010-0110
- [ ] OMB Control Number 1010-0074
- [ ] OMB Control Number 1010-0088
- [ ] OMB Control Number 1010-0138
- [ ] OMB Control Number 1010-0104
- [ ] OMB Control Number 1010-0095
- [ ] OMB Control Number 1010-0075
- [ ] OMB Control Number 1010-0061
- [ ] OMB Control Number 1010-0113
- [ ] OMB Control Number 1010-0135
- [ ] OMB Control Number 1010-0129
- [ ] OMB Control Number 1010-0126
- [ ] OMB Control Number 1010-0042
- [ ] OMB Control Number 1010-0121
- [ ] OMB Control Number 1010-0123
- [ ] Forms
- [ ] Reference Lists and Codes
- [ ] Handbooks
- [ ] Minerals Revenue Reporter Handbook
- [ ] Minerals Production Reporter Handbook
- [ ] Solid Minerals Reporter Handbook
- [ ] Geothermal Payor Handbook - Class 1 Leases
- [ ] Geothermal Payor Handbook - Class 2 & 3  Leases
- [ ] Indian Payor Handbook
- [ ] Valuation and Pricing
- [ ] Interactive Example (2016 Valuation Rule Federal Processed Gas Reporting Example)
- [ ] Interactive Example (Percent-of-Proceeds (POP) Contract Reporting Pre-2016)
- [ ] Reporter Letters
- [ ] Reporter Training
- [ ] Paying (Home)
- [ ] Payment Options
- [ ] Late Payment Interest
- [ ] Unpaid Debt
- [ ] Compliance & Enforcement (Home)
- [ ] Compliance 
- [ ] Enforcement & Litigation Support
- [ ] Appeals
- [ ] Cooperative Agreements (STRAC)
- [ ] Indian Resources (Home)
- [ ] Assistance
- [ ] Tribal Consultation
- [ ] Indian Pricing
- [ ] Indian References
- [ ] Tribal Lockbox Payments
- [ ] About ONRR
- [ ] Public Affairs
- [ ] Organization & Leadership
- [ ] Office Locations
- [ ] Employment
- [ ] Engagement
- [ ] Indian Oil Valuation Negotiated Rulemaking Committee
- [ ] Interagency Coordination
- [ ] Freedom of Information Act
- [ ] Anti-Harassment Program
- [ ] Contact Us
- [ ] Search Results
- [ ] Getting Started
- [ ] System Access
- [ ] Leases & Agreements
- [ ] Oil & Gas Production
- [ ] Oil & Gas Royalty
- [ ] Oil & Gas Rent
- [ ] Solid Minerals
- [ ] Geothermal
- [ ] Payments & Debt Collection
- [ ] Updating Contact Info
- [ ] Valuation & Pricing
- [ ] Compliance & Enforcement
- [ ] Indian Services
- [ ] Contacts by Company
- [ ] Information Requests
- [ ] Events
- [ ] Glossary
- [ ] Sitemap
- [ ] Disclaimer

**Comment by cthomasONRR** [comment]
@lpgoldstein @mcharg I'm noticing that all pages so far are getting the following errors. (And highest accessibility rating is 84%.) Should we try to fix the following errors before running any more reports?

Doesn't like the left-side navigation pane:
![image](https://user-images.githubusercontent.com/72526208/171222268-d989f888-96fd-4946-8037-28957a55c011.png)

Doesn't seem to ID the alt text with the hero image:
![image](https://user-images.githubusercontent.com/72526208/171222562-9088812e-39ad-409d-a2ae-1cdaef9ab9a2.png)

No alt text for social media icons:
![image](https://user-images.githubusercontent.com/72526208/171222758-289da54e-aa7b-4197-a36c-994ef5b59cbc.png)

**Comment by cthomasONRR** [comment]
Also, we need to discuss if we want to check all individual tabs... if so, there will be way more line items to add to the list above.

**Comment by cthomasONRR** [comment]
@mcharg @lpgoldstein 

I got all pages tested except for: 

- search results (main search not working), 
- Glossary (that collection is not yet uploaded), and 
- Contacts by Company (could not find this page). 

Did the contacts by company page get removed?

**Comment by mcharg** [comment]
@cthomasONRR I removed the link to contacts by company, since we can launch without it.

**Comment by mcharg** [comment]
I made sure we have issues for all the errors found in the Lighthouse reports.
